ethereum/vyper
# Use mixedCase for public functions/variables in examples/docs

### - What I did

Update Vyper examples/documentation so public functions/variables use mixedCase instead of snake_case. 

Stems from discussion in #905 

### - How I did it

Meticulous find & replace while running tests 😄 

### - How to verify it

- Running `python setup.py test` on the examples folder shows the examples are working
- Looking over docs to ensure all references have been updated

### - Description for the changelog

None.

### - Cute Animal Picture

![image](https://user-images.githubusercontent.com/4423365/47260026-67137c00-d481-11e8-85e7-fad26f2cd440.png)
